lib/pharos/host/debian/debian.rb

```diff
@@ -54,6 +49,17 @@ def configure_firewalld
       def reset
         exec_script("reset.sh")
       end
+
+      def configure_repos
+        host_repositories.each do |repo|
+          repo_path = "/etc/apt/sources.list.d/#{repo.name}"
+          next if transport.file(repo_path).exist?
+
+          transport.exec!("curl -fsSL #{repo.key} | apt-key add -")
```

Comment:
Should it maybe be something like `key_url`?

Reply:
👍 make sense. also, needs to be conditional call to add it, key is not really necessary in all cases